requirements.txt

```diff
@@ -6,3 +6,4 @@ pyasn1==0.4.8
 python-hosts==1.0.0
 pyOpenSSL==19.1.0
 requests==2.23.0
+urllib3==1.26.5
```

Comment:
Why do we need it here? It seems like urllib3 is installed by another python library / package and we should let it use the version it installs.

Reply:
Hello @danishfoxpass , exactly Snyk mention that the current urllib3 version that is being installed as a dependecy for another package has a vulnerability and we should update that  version, evidence below
```
Introduced through: project@0.0.0 › boto3@1.9.195 › botocore@1.12.253 › urllib3@1.25.11
Remediation: Pin urllib3 to version 1.26.5
```